chart/test/k3d-nightly-ci.yaml

```diff
@@ -0,0 +1,9 @@
+# k3s overrides for nightly ci
+# ----------------------------
+
+# don't pull use, existing images
+backend_pull_policy: "Never"
+frontend_pull_policy: "Never"
+
+
+invite_expire_seconds: 10
```

Comment:
If we move the other settings to base test.yaml, might not need this file at all?
Can also do: `--set invite_expire_seconds=10` via command-line, or just make this the default as well?

Reply:
Makes sense! Latest commit uses `--set invite_expire_seconds=10`. I'd prefer not to set it as the default for all tests because we have some tests that poll the pending invite endpoints and I don't want this setting to cause intermittent failures for those tests.